packages/remirror__extension-link/src/link-extension.ts

```diff
@@ -332,7 +332,7 @@ export class LinkExtension extends MarkExtension<LinkOptions> {
     return [
       {
         type: 'mark',
-        regexp: /https?:\/\/(www\.)?[\w#%+.:=@~-]{2,256}\.[a-z]{2,8}\b([\w#%&+./:=?@~-]*)/gi,
+        regexp: this.options.autoLinkRegex,
```

Comment:
The regex used here has the `g` and `i` flags

We should add them to the default `autoLinkRegex` so not to be breaking